Sasquatch/Sasquatch/ViewControllers/MSDistributeViewController.swift

```diff
@@ -2,16 +2,67 @@
 // Licensed under the MIT License.
 
 import UIKit
+import AppCenterDistribute
 
 class MSDistributeViewController: UITableViewController, AppCenterProtocol {
 
   @IBOutlet weak var enabled: UISwitch!
   @IBOutlet weak var customized: UISwitch!
+  @IBOutlet weak var preStartSwitch: UISwitch!
+  @IBOutlet weak var updateTrackField: UITextField!
   var appCenter: AppCenterDelegate!
-  
+
+  enum UpdateTrack: String {
+    case Public = "Public"
+    case Private = "Private"
+
+    var state: MSUpdateTrack {
+       switch self {
+       case .Public: return .public
+       case .Private: return .private
+       }
+    }
+
+    static func getSelf(by track: MSUpdateTrack) -> UpdateTrack {
+       switch track {
+       case .public: return .Public
+       case .private: return .Private
+       }
+    }
+
+    static let allValues = [Public, Private]
+  }
+
+  private var updatePicker: MSEnumPicker<UpdateTrack>?
+  private var updateTrack = UpdateTrack.Public {
+    didSet {
+       self.updateTrackField.text = self.updateTrack.rawValue
+    }
+  }
+
   override func viewDidLoad() {
     super.viewDidLoad()
     self.customized.isOn = UserDefaults.init().bool(forKey: kSASCustomizedUpdateAlertKey)
+    self.preStartSwitch.isOn = UserDefaults.standard.value(forKey: kMSUpdateTrackBeforeStartValue) != nil
```

Comment:
Like on other platforms, the original feedback was to replace check box by a picker.